src/shared/ui/modal/ModalProvider.tsx

```
@@ -0,0 +1,75 @@
+    return id;
+  }, []);
+
+  const close = useCallback((id: string) => {
+    setModals((prev) => {
+      prev.find((m) => m.id === id)?.onClose?.();
+      return prev.filter((m) => m.id !== id);
+    });
+  }, []);
+
+  const closeAll = useCallback(() => {
+    modals.forEach((m) => m.onClose?.());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)